src/middleware/routeHandler.ts

```diff
@@ -108,16 +108,19 @@ export const attachWriteRouteHandlerWrapper: RouteWrapper<{
     return
   }
 
-  Promise.resolve(routeHandler(req, res, next)).catch(async (err: Error) => {
-    await unlock(siteName)
-    next(err)
```

Comment:
could we just log on next? that way we can monitor directly on prod if any of the route handlers call next before we do?